networking/ciao-cnci-agent/client.go

```diff
@@ -420,6 +429,39 @@ func discoverUUID() (string, error) {
 	return metaData.UUID, nil
 }
 
+//Rebuild network state from database
+func rebuildNetworkState(db *cnciDatabase) error {
+	var lastError error
+	if db == nil {
+		return nil
+	}
+
+	db.SubnetMap.Lock()
+	defer db.SubnetMap.Unlock()
+	db.PublicIPMap.Lock()
+	defer db.PublicIPMap.Unlock()
+
+	for key, subnet := range db.SubnetMap.m {
+		glog.Infof("Key: %v Subnet: %v", key, subnet)
+		err := addRemoteSubnet(subnet)
+		if err != nil {
+			lastError = err
+			glog.Errorf("rebuildNetworkState: %v", err)
```

Comment:
@mcastelino Ack, in which case this is great! If the error doesn't need any more context then there is no need for a wrap.